web_app/dist/main.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+        
+    </head>
+ 
+    <body>  
+        
+    <div id="sticky-header">
+        <div class="navbar">
+            <img id="logo" src="images/icon_small.png" alt="TaskViewer Logo">
+            <a>Log out</a>
+            <a>Sort</a>
+            <a>Filter by</a>
+            <input id="search-bar" type="text" placeholder="Search..">
+        </div>
+        <div class="new-flex-row-anchor-right">
+            <button id="create-task-button"class="button">+</button>    
```

Comment:
Add space in between `id` and `class` attributes

Reply:
done (just line 24 yeah?)